docs/code-of-conduct.md

```diff
@@ -95,6 +96,7 @@ Harassment includes:
 The toxic behavour includes:
 * unreasonable repetitive critic of the school, its program, mentors and students
 * rude language outside of random channel
+* insults of other school members in any available communication channels
```

Comment:
mb change `insults` to 'abuse'?

Reply:
I'm not that fluent, just B2 level, but to me it seems that insult is exact what we were faced in yesterday accident. I guess, affront can be added as well, as an alternative.